src/content-helper/dashboard-page/pages/traffic-boost/sidebar/sidebar.scss

```
@@ -15,7 +15,8 @@
 	background: var(--white);
 	border-radius: var(--grid-unit-10);
 	border-bottom: 1px solid var(--gray-200);
-	height: calc(100vh - to_rem(65px));
+	// Fallback is 0px because Core only defines the property when the bar renders.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)